### PR TITLE
[Security certifications] - PR to merge to master

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- This is the PR to merge all changes from security certifications into master. The PR was initially reviewed against master [here](https://github.com/canonical-web-and-design/ubuntu.com/pull/9880), but it was from my fork instead of the `canonical-web-and-design:security-certifications`.

## QA

Check nav, for any broken links, spelling etc

### CODE REVIEW

**Comment by carkod** [comment]
@bethcollins92 please rebase when you can, let us know if you need help

**Comment by nmav** [comment]
@bethcollins92 is the demo at https://ubuntu-com-10011.demos.haus/security/certifications ? I still see the first version of the pages

**Comment by bethcollins92** [comment]
> @bethcollins92 is the demo at https://ubuntu-com-10011.demos.haus/security/certifications ? I still see the first version of the pages

@nmav yeah, can you try and hard refresh your page the demo could be cached

**Comment by bethcollins92** [comment]
@SirSamTumless All the work in this feature branch has already been code and qa +1 already. This is the final check before we merge to master. Once we have the PdM +1 it's good to go

**Comment by nmav** [comment]
Actually it was browser caching problem works now. Only one comment:
 - docs should not be an option in the top bar; we are only referring explicitly to that

File: templates/security/fips.html [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+      </thead>
+      <tbody>
+        <tr>
+          <th>Linux Kernel Crypto API</th>
+          <td>
+            <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/2962">#2962</a>
+            <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/3724">#3724</a>
+          </td>
+          <td>
+            <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/3647">#3647</a>, <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/3664">#3664</a> (AWS),<br><a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/3683">#3683</a> (Azure)
```
Comment (nmav) [inline templates/security/fips.html:148]: 3664 is repeated twice

File: templates/security/fips.html [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+          <th>Ubuntu 16.04 LTS<br>on x86-64, IBM Power8 and IBM Z</th>
+          <th>Ubuntu 18.04 LTS<br>on x86-64 and IBM Z</th>
+          <th>Ubuntu 20.04 LTS<br>on x86-64 and IBM Z</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <th>Linux Kernel Crypto API</th>
+          <td>
+            <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/2962">#2962</a>
```
Comment (nmav) [inline templates/security/fips.html:144]: no comma between these two

File: templates/security/fips.html [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+            <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/3632">#3632</a>
+          </td>
+          <td>
+            <p>Merged with OpenSSL</p>
+          </td>
+        </tr>
+        <tr>
+          <th>OpenSSL</th>
+          <td>
+            <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/2888">#2888</a>
```
Comment (nmav) [inline templates/security/fips.html:182]: no comma between these two

File: templates/security/fips.html (hunk 1 above)
Comment (nmav) [reply templates/security/fips.html:148]: No comma between Azure and GCP

**Comment by carkod** [comment]
> Actually it was browser caching problem works now. Only one comment:
> 
> * docs should not be an option in the top bar; we are only referring explicitly to that
> 
> Other than that, looks good to go

Sorry, what do you mean? do you want to remove docs from the navigation?

**Comment by nmav** [comment]
> > Actually it was browser caching problem works now. Only one comment:
> > 
> > * docs should not be an option in the top bar; we are only referring explicitly to that
> > 
> > Other than that, looks good to go
> 
> Sorry, what do you mean? do you want to remove docs from the navigation?

yes, no need to get through navigation to docs. We have direct links from the pages.

**Comment by bethcollins92** [comment]
Thanks @nmav, made those changes. If everything is good to go can you add the PdM +1 label please?

**Review by nmav — APPROVED** [review]

**Comment by nmav** [comment]
> Thanks @nmav, made those changes. If everything is good to go can you add the PdM +1 label please?
> 
> Screenshot 2021-07-19 at 17 02 38

I do not have access to edit labels but +1 from me.

**Comment by anthonydillon** [comment]
Sorry, @nmav I have bumped your permissions.

**Comment by nmav** [comment]
Thank you, great work!

**Pull request merged**